playbooks/roles/validation/tasks/main.yml

```diff
@@ -8,3 +8,8 @@
   fail:
     msg: "tinyproxy cannot be enabled if ssh forward user is disabled"
   when: not streisand_ssh_forward_enabled and streisand_tinyproxy_enabled
+
+- name: Validate that the maximum number of clients is set to a reasonable amount
+  fail:
+    msg: "Unreasonable number of VPN clients sepcified"
```

Comment:
I think this might be clearer as "Too many VPN clients specified. vpn_clients must be greater than 0 and less than 20".